espresso/src/com.oracle.truffle.espresso/src/com/oracle/truffle/espresso/nodes/BytecodeNode.java

```diff
@@ -2136,11 +2136,9 @@ private void putPoolConstant(VirtualFrame frame, int top, char cpi, int opcode)
         } else if (constant instanceof StringConstant) {
             assert opcode == LDC || opcode == LDC_W;
             StaticObject internedString = pool.resolvedStringAt(cpi);
-            Meta meta = getMeta();
-            StaticObject obj = meta.toGuestString(meta.toHostString(internedString));
```

Comment:
@fhowar I don't know why you added this and whether it is really necessary.
The constant op code does not need it and all other exceptions should be handled in the `markObjectWithIfTaint` method, IMHO.